Tone/instrument/CymbalSynth.js

```diff
@@ -0,0 +1,304 @@
+		this.inharmRatios = [1.0, 1.483, 1.932, 2.546, 2.630, 3.897];
+
+		/**
+		 *  Harmonic ratio that produces a more pitched cymbal
+		 *  @private
+		 *  @static
+		 *  @type {Array}
+		 */
+		this.harmRatios = [1.0, 1.5, 2.025, 2.975, 4.0, 6.0];
+
+		/**
+		 *  A series of FMSynths
+		 *  @private
+		 *  @type {Array}
+		 */
+		this._oscillators = [];
+
+		/**
+		 *  Tone.ScaledSignals used for interpolating between harmonic
+		 *  and inharmonic ratios
+		 *  @private
+		 *  @type {Array}
+		 */
+		this._scaledSignals = [];
+
+		/**
+		 *  Tone.Multipliers for frequency multiplication of scaled signals
+		 *  @private
+		 *  @type {Array}
+		 */
+		this._freqMult  = [];
+
+		//make all the FMSynths, connect the frequency signals
+		for(var i = 0; i < 6; i++){
+			this._scaledSignals[i] = new Tone.Scale(this.inharmRatios[i], this.harmRatios[i]);
+			this._freqMult[i] = new Tone.Multiply();
+			this._oscillators[i] = new Tone.SimpleFM({
+				"harmonicity" : 1.5,
+				"modulationIndex" : 10,
+				"carrier" : {
+					"oscillator" : {
+						"type" : "square"
+					}
+				},
+				"modulator" : {
+					"oscillator" : {
+						"type" : "pulse",
+						"width" : 0.75
+					}
+				},
+			});
+			this.harmonicity.connect(this._scaledSignals[i]);
+			this._scaledSignals[i].connect(this._freqMult[i], 0, 0);
+			this.frequency.connect(this._freqMult[i], 0, 1);
+			this._freqMult[i].connect(this._oscillators[i].frequency);
+			this._oscillators[i].connect(this.strike);
+			this._oscillators[i].connect(this.body);
+			this._oscillators[i].triggerAttack(Tone.context.currentTime);
+		}
+	};
+
+	Tone.extend(Tone.CymbalSynth, Tone.Instrument);
+
+	/**
+	 *  the default values
+	 *  @static
+	 *  @const
+	 *  @type {Object}
+	 */
+	Tone.CymbalSynth.defaults = {
+		"harmonicity" : 0,
+		"frequency" : 200,
+		"strike" : {
+			"volume" : 0,
+			"cutoff" : 3500, 
+			"resonance" : 6,
+			"envelope": { 
+				"attack" : 0.01, 
+				"decay" : 0.25
+			}
+		},
+		"body" : {
+			"volume" : 0,
+			"cutoff" : 7000,
+			"resonance" : 6,
+			"envelope" : {
+				"attack" : 0.01,
+				"decay" : 1.5
+			}
+
+
+		},
+	};
+
+	/**
+	 *  Trigger the cymbal at a given time
+	 *  @param  {Time} [time=now]     the time, if not given is now
+	 *  @param  {Number} [velocity = 1] velocity defaults to 1
+	 *  @returns {Tone.Drumsynth} this
+	 *  @example
+	 *  cymbal.triggerAttack("0:1:1". 0.75);
+	 */
+	Tone.CymbalSynth.prototype.triggerAttack = function(time, velocity){
+		time = this.toSeconds(time);
+		this.body.envelope.triggerAttack(time, velocity);
+		this.strike.envelope.triggerAttack(time, velocity);
+		return this;
+	};
+
+	/**
+	 *  Trigger the release of the cymbal.
+	 *  Can be used to [choke](https://en.wikipedia.org/wiki/Cymbal_choke) 
+	 *  the cymbal
+	 *  @param  {Time} [time=now] the time the cymbal will release
+	 *  @returns {Tone.CymbalSynth} this
+	 *  @example
+	 *  //set the release to a small amount for a realistic choke
+	 *  cymbal.body.envelope.release = 0.2;
```

Comment:
I might switch the default envelope release to something like 0.2 instead of 0. That way `triggerRelease` is useful right away without having to change any settings. Should probably put envelope release in the defaults then as well.